app/views/assessments/new.html.erb

```diff
@@ -65,17 +65,17 @@
             </div>
           </div>
 
-          <div class="field">
-            <span>Assessment Type</span>
+          <fieldset class="field-section">
+            <legend class="field-section__title">Assessment Category</legend>
```

Comment:
The figma designs say "Assessment Type", but the field is 'category' and we are referring to it as category elsewhere in the app - makes sense to me to keep category here for the time being